### ISSUE TITLE
Recursive pylintrc includes?

### ISSUE THREAD

**Issue by rhelmot** [PyCQA/pylint#2211]
I've been trying to set up a pylintrc which includes another pylintrc from outside the current python project. It doesn't seem to be actually using the `rcfile` directive that `--generate-rcfile` conveniently informed me of.

To reproduce this bug (...I am assuming this is a bug), use [this tarball](https://github.com/PyCQA/pylint/files/2121626/reproduce.tar.gz) and run `pylint c.py` inside the `b` folder. The expected behavior is that both the `eval-used` and `unused-argument` messages should be disabled, but what happens is that `eval-used` is still reported.

Version information:

```
pylint 1.6.4, 
astroid 1.4.8
Python 2.7.14 (default, Sep 23 2017, 22:06:14) 
[GCC 7.2.0]
```

**Comment by PCManticore** [comment]
This is currently not supported. The work that Ashley's been doing for per directory configuration will have a solution for this kind of problem, but as of right now we don't support the concept of recursive configurations.

https://github.com/PyCQA/pylint/projects/1
